google_memcache/README.md

```diff
@@ -0,0 +1,46 @@
+# Terraform Module: Memcache
+Creates a memcache instance within GCP using Cloud Memorystore
+
+## Example
+
+```hcl
+locals {
+  name     = "test-memcache"
+  realm    = "nonprod"
+  networks = try(data.terraform_remote_state.vpc.outputs.networks.realm[local.realm], {})
+}
+
+module "memcache" {
+  source = "github.com/mozilla/terraform-modules//google_memcache?ref=main"
+
+  application        = local.name
+  environment        = "dev"
+  realm              = local.realm
+  memory_size_mb     = 2048
+  authorized_network = local.subnetworks.regions["us-west1"]["network"]
```

Comment:
Not sure if we want to update the example to match the one here https://github.com/mozilla-it/global-platform-admin/pull/73

Reply:
Good catch, forgot to update that section here. Fixed.